tests/src/Kernel/Controller/ViewerControllerTest.php

```diff
@@ -121,11 +159,16 @@ public function testEditorNoViewer(): void {
       $this->assertBadRequestResponse(
         'The Collabora Online editor/viewer is not available.',
         $request,
+        $name
+      );
+      $this->assertLogMessage(
+        RfcLogLevel::WARNING,
+        "Cannot show the viewer/editor.<br>\n" . Error::DEFAULT_ERROR_MESSAGE,
         [
-          'message' => "Cannot show the viewer/editor.<br>\n" . Error::DEFAULT_ERROR_MESSAGE,
-          'level' => RfcLogLevel::WARNING,
+          '%type' => CollaboraNotAvailableException::class,
+          '@message' => '',
```

Comment:
Given that we assert the exception message, we could create the CollaboraNotAvailableException with a proper message in the transcoder mock above.
Also, actually, right now the transcoder is documented to throw CollaboraJwtKeyException, which extends CollaboraNotAvailableException.
It's not a blocker, I would say.